server/mergin/sync/public_api_controller.py

```diff
@@ -813,7 +816,9 @@ def project_push(namespace, project_name):
         if not all(ele.path != item.path for ele in project.files):
             abort(400, f"File {item.path} has been already uploaded")
         if not is_valid_path(item.path):
-            abort(400, f"File {item.path} contains invalid characters.")
+            abort(400, f"File '{item.path}' contains invalid characters.")
```

Comment:
How about
```suggestion
            abort(400, f"Unsupported file name detected: {item.path}")
```
to make it clear it relates to the file name(path), not the content of the file